pom.xml

```diff
@@ -559,12 +492,34 @@ THE SOFTWARE.
                   <arguments>build</arguments>
                 </configuration>
               </execution>
-
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>yarn-ci-lint</id>
+      <activation>
+        <property>
+          <name>env.CI</name>
+        </property>
+        <file>
+          <exists>package.json</exists>
+        </file>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.github.eirslett</groupId>
+            <artifactId>frontend-maven-plugin</artifactId>
+            <version>1.15.0</version>
+            <executions>
               <execution>
-                <id>yarn lint</id>
+                <id>yarn lint:ci</id>
                 <goals>
                   <goal>yarn</goal>
                 </goals>
+                <phase>test</phase>
```

Comment:
Needed in order to get CI builds to honor https://github.com/jenkinsci/jenkins/blob/57d8fa0b72c0f88f1e7eece8fb0bd5a2a0d8d0ea/Jenkinsfile#L96 to address https://issues.jenkins.io/browse/JENKINS-71021?focusedId=448486&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-448486 (cf. https://github.com/eirslett/frontend-maven-plugin/blob/f39b40c20971ecb810adce465d37f28d9dc4c0e6/frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/AbstractFrontendMojo.java and also https://github.com/jenkinsci/plugin-pom/issues/874).